package.json

```diff
@@ -12,6 +12,12 @@
   "scripts": {
     "test": "mocha tests/**/*-test.js"
   },
+  "files": [
```

Comment:
https://docs.npmjs.com/files/package.json#files

> Certain files are always included, regardless of settings:
>
> package.json
> README
> CHANGES / CHANGELOG / HISTORY
> LICENSE / LICENCE
> NOTICE
> The file in the "main" field

I guess you really only need "yarn.lock"? Weird.

Reply:
hmm yeah looks like it, I was referencing the express package.json but looks like we only need index.js and yarn.lock

Reply:
and index.js is the "main" file right? so technically that's not needed. tho i see no harm in including it

Reply:
ohhhhh I see yeah that's interesting, might as well take it out if it's not necessary

Reply:
Why include yarn.lock?

Reply:
My assumption was we'd always want to include it so installing the package would always have the right dependency versions? But it looks like they actually auto-ignore other lock files, so maybe yarn.lock should be ignored as well:
![image](https://user-images.githubusercontent.com/10302274/34057875-bd06fda4-e18d-11e7-8c89-a40c0c40cac1.png)